validation/invalid/ttml2-invld-xlink-href-nested-image.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- invalid - xlink:href - nested link is not permitted -->
+<tt xml:lang="en" xmlns="http://www.w3.org/ns/ttml"
+    xmlns:ttp="http://www.w3.org/ns/ttml#parameter" ttp:profile="ttml2-transformation"
+    xmlns:xlink="http://www.w3.org/1999/xlink"
+    xmlns:ttva="http://skynav.com/ns/ttv/annotations" ttva:warnOn="all" ttva:expectedErrors="1" ttva:expectedWarnings="0" ttva:model="ttml2">
+  <head/>
+<body>
+  <div>
+    <p>
+      <span xlink:href="https://example.com/">
+        <image src="../resources/test.png" type="image/png" xlink:href="https://example.com/"/>
```

Comment:
Previously, and still with this new test, we only validate that nested _child_ elements do not specify an xlink, but the spec prohibits any _descendant_ elements from specifying them. We never had a test for this before but arguably we should have done. This can easily be achieved by making the following suggested change.

```suggestion
    <p xlink:href="https://example.com/">
      <span>
        <image src="../resources/test.png" type="image/png" xlink:href="https://example.com/"/>
```

Reply:
I'm going to defer this suggestion for future consideration in 3e TS or later.